addons/account/wizard/accrued_orders.py

```diff
@@ -0,0 +1,193 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from dateutil.relativedelta import relativedelta
+from collections import defaultdict
+import json
+from odoo import models, fields, api, _, Command
+from odoo.tools import format_date
+from odoo.exceptions import UserError
+
+
+class AccruedExpenseRevenue(models.TransientModel):
+    _name = 'account.accrued.orders.wizard'
+    _description = 'Accrued Orders Wizard'
+
+    def _get_account_domain(self):
+        if self.env.context.get('active_model') == 'purchase.order':
+            return [('user_type_id', '=', self.env.ref('account.data_account_type_current_liabilities').id)]
+        else:
+            return [('user_type_id', '=', self.env.ref('account.data_account_type_current_assets').id)]
+
+    company_id = fields.Many2one('res.company', default=lambda self: self.env.company)
```

Comment:
specify the `comodel_name` keyword plz

Reply:
also, `company_id` should be computed as well base on the business models you are working with so